requirements.txt

```diff
@@ -13,5 +13,3 @@ pytmc>=2.7.0
 pyyaml
 schema
 scipy
-sqlalchemy<2.0.0
```

Comment:
For whatever reason this isn't being picked up by our pip tests?...
https://github.com/pcdshub/pcdsdevices/actions/runs/5524933875/jobs/10077909012?pr=1142#step:10:135

Reply:
Some other package must still be constraining them

Reply:
aha https://github.com/pcdshub/pcdscalc/blob/master/requirements.txt

Reply:
And the conda recipe in pcdshub/pcdscalc is still pinned as well, but the conda feedstock is not.  

So many recipes